README.md

```diff
@@ -4,37 +4,61 @@
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
-## Running the website locally using Hugo
+# Using this repository
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run the website locally using Hugo, or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 
-Before building the site, clone the Kubernetes website repository:
+## Prerequisites
 
-```bash
+To use this repository, you need the following installed locally:
+
+- [yarn](https://yarnpkg.com/)
+- [npm](https://www.npmjs.com/)
+- [Go](https://golang.org/)
+- [Hugo](https://gohugo.io/)
+- A container runtime, like [Docker](https://www.docker.com/).
+
+Before you start, install the dependencies. Clone the repository and navigate to the directory:
+
+```
 git clone https://github.com/kubernetes/website.git
 cd website
-git submodule update --init --recursive --depth 1
 ```
 
-**Note:**  The Kubernetes website deploys the [Docsy Hugo theme](https://github.com/google/docsy#readme).
-If you have not updated your website repository, the `website/themes/docsy` directory is empty. The site cannot build
-without a local copy of the theme.
+The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
 
-Update the website theme:
+```
+# install dependencies
+yarn
 
-```bash
+# pull in the Docsy submodule
 git submodule update --init --recursive --depth 1
 ```
 
+## Running the website using a container
+
+To build the site in a container, run the following to build the container image and run it:
+
+```
+make container-image
+make container-serve
+```
+
+Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+
+## Running the website locally using Hugo
+
+Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+
 To build and test the site locally, run:
 
 ```bash
-hugo server --buildFuture
+make serve
```

Comment:
I'm not opposed to this, but I think the original is more OS compatible (... Windows :joy: )

Reply:
I agree. What is the reasoning for changing to `make serve`?